### PR TITLE
Add button refresh and remove automatic refresh

### PR DESCRIPTION
**Pull request by kwunkiuma**
The signup sheet no longer refreshes upon someone signing on/off the event.

A refresh button has been added to the signup sheet to manually refresh the sheet.
image

### CODE REVIEW

File: DiscordBot/Program.cs [hunk 1]
```diff
@@ -88,29 +86,36 @@ private static async Task MainAsync()
...
+                case (":arrows_counterclockwise:"):
+                    await UpdateSignupSheet(eventArguments, discordEvent, eventsSheetsService);
+                    break;
+                case (":no_entry_sign:"):
+                    await eventsSheetsService.ClearResponsesForUserAsync(discordEvent.Key, eventArguments.User.Id);
+                    await client.SendMessageAsync(dmChannel, $"You've signed off {discordEvent.Name}.");
+                    break;
+                default:
+                    try
+                    {
```
Comment (FridaTveit) [inline DiscordBot/Program.cs:100]: This default case might be worth splitting out into its own function to reduce nesting? 🙂

**Review by FridaTveit — COMMENTED** [review]
One suggestion otherwise looks good 🙂

**Comment by kwunkiuma** [comment]
Also renamed UpdateSignupSheet to UpdateSignupMessage.

File: DiscordBot/Program.cs [hunk 2]
```diff
@@ -89,36 +90,49 @@ private static async Task MainAsync()
...
+            try
+            {
+                await eventsSheetsService.AddResponseForUserAsync(
+                    discordEvent.Key,
+                    eventArguments.User.Id,
+                    eventArguments.Emoji.GetDiscordName()
+                );
+            }
+            catch (ResponseNotFoundException)
+            {
```
Comment (FridaTveit) [inline DiscordBot/Program.cs:126]: You want to return here I think? 🙂

**Review by FridaTveit — COMMENTED** [review]
One small thing 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**